using_images/other_images/jenkins.adoc

```diff
@@ -64,6 +79,71 @@ $ oc new-app -e \
     openshift/jenkins-1-centos7
 ----
 
+If `*JENKINS_PASSWORD*` is not specified on the `oc new-app` invocation, the password is an auto-generated, random string.  It will be reported in the new-app
```

Comment:
this isn't accurate.  the randomly password behavior only applies when instantiating the jenkins template.  this is just using new-app to stand up the image directly, not via the template, so the parameter generation logic is not applicable.